pkg/server/service.go

```diff
@@ -128,6 +128,10 @@ func NewCRIService(config criconfig.Config, client *containerd.Client) (CRIServi
 		selinux.SetDisabled()
 	}
 
+	if c.client.SnapshotService(c.config.ContainerdConfig.Snapshotter) == nil {
+		return nil, errors.Errorf("snapshotter %q does not exist", c.config.ContainerdConfig.Snapshotter)
```

Comment:
can we normalize the errors in this func? maybe defer a wrap for  "Failed to initialize cni"? Because some say cni failed some say what failed..

Reply:
Done.